src/hotspot/cpu/aarch64/interp_masm_aarch64.cpp

```diff
@@ -1880,3 +1881,19 @@ void InterpreterMacroAssembler::load_resolved_indy_entry(Register cache, Registe
   add(cache, cache, Array<ResolvedIndyEntry>::base_offset_in_bytes());
   lea(cache, Address(cache, index));
 }
+
+void InterpreterMacroAssembler::load_field_entry(Register cache, Register index, int bcp_offset) {
+  // Get index out of bytecode pointer
+  get_cache_index_at_bcp(index, bcp_offset, sizeof(u2));
+  // Take shortcut if the size is a power of 2
+  if (is_power_of_2(sizeof(ResolvedFieldEntry))) {
+    lsl(index, index, log2i_exact(sizeof(ResolvedFieldEntry))); // Scale index by power of 2
+  } else {
+    mov(cache, sizeof(ResolvedFieldEntry));
+    mul(index, index, cache); // Scale the index to be the entry index * sizeof(ResolvedInvokeDynamicInfo)
```

Comment:
A few cosmetic suggestions: 

```suggestion
    mul(index, index, cache); // Scale the index to be the entry index * sizeof(ResolvedFieldEntryInfo)
```

you might want to update this comment in x86 file as well ;-) 

Beside this I guess you should update header-year for `src/hotspot/share/ci/ciStreams.cpp` , `src/hotspot/share/interpreter/interpreterRuntime.hpp` , `test/hotspot/gtest/oops/test_cpCache_output.cpp`.